src/node/config.ts

```diff
@@ -73,6 +73,9 @@ export async function resolveConfig(
     })
   const site = await resolveSiteData(root, userConfig)
   const srcDir = normalizePath(path.resolve(root, userConfig.srcDir || '.'))
+  const assetsDir = userConfig.assetsDir
+    ? userConfig.assetsDir.replace(/\//g, '')
```

Comment:
@zonemeen Do you remember why this replace statement was added?

Reply:
Sorry for the late reply. I was only thinking about the top-level assetsDir configuration and not about supporting nested assetsDir, and then I added this regular rule to remove any slash at the beginning or end that a user might add to a customized assetsDir name.